aiofreepybox/api/connection.py

```diff
@@ -0,0 +1,22 @@
+class Connection:
+
+    def __init__(self, access):
+        self._access = access
+
+    async def get_config(self):
```

Comment:
This should be called `get_status`; the `get_config` should get `connection/config`. You could optionally add `set_config` also, if you need it.

Reply:
I changed the get config to connection/config/ and added a get_status for connection/